src/Public/Authentication.ps1

```diff
@@ -1513,9 +1527,14 @@ function Add-PodeAuthMiddleware {
 
         [Parameter()]
         [string]
-        $Route
+        $Route,
+
+        [string[]]
+        $DefinitionTag
```

Comment:
Similar to the Route one, would benefit from being called `OADefinitionTag`. Just so it's obvious it's related to OpenAPI.

Reply:
I agree